torchgeo/datasets/usavars.py

```diff
@@ -0,0 +1,223 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""USAVars dataset."""
+
+import glob
+import os
+from typing import Any, Callable, Dict, List, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import rasterio
+import torch
+from matplotlib.figure import Figure
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import download_url, extract_archive
+
+
+class USAVars(VisionDataset):
+    """USAVars dataset.
+
+    Dataset format:
+    * images are 4-channel tifs
+    * labels are singular float values
+
+    Dataset labels:
+    - tree cover
+    - elevation
+    - population density
+    - nighttime lights
+    - income per houshold
+    - road length
+    - housing price
+
+    .. versionadded:: 0.3
+    """
+
+    csv_prefix = (
+        "https://files.codeocean.com/files/verified/"
+        + "fa908bbc-11f9-4421-8bd3-72a4bf00427f_v2.0/data/int/applications/"
+    )
+    csv_postfix = "_CONTUS_16_640_POP_100000_0.csv?download"
+
+    data_url = "https://mosaiks.blob.core.windows.net/datasets/uar.zip"
+    dirname = "usavars"
+    zipfile = dirname + ".zip"
+
+    md5 = "677e89fd20e5dd0fe4d29b61827c2456"
+
+    label_urls = {
+        "housing": csv_prefix + "housing/outcomes_sampled_housing" + csv_postfix,
```

Comment:
You could do these with f-strings

```suggestion
        "housing": f"{csv_prefix}housing/outcomes_sampled_housing{csv_postfix}",
```